### DISCUSSION TITLE
with UI opened, I cannot see the lines connecting these notes

### DISCUSSION THREAD

**Discussion by 565514246** [comfyanonymous/ComfyUI#5612] [Q&A] [answered]
Hi everyone, I had a problem: cannot see the lines connecting these notes. I downloaded UI_Windows_portable, and extract it to my desktop, after running the nvidia_gpu, in the main windows I got basically no error message, only a warning as below (I deleted some part of the scripts shown): 
([START] Security scan
[DONE] Security scan
## ComfyUI-Manager: installing dependencies done.
** ComfyUI startup time: 2024-11-14 12:26:09.967615
** Platform: Windows
** **Python version: 3.12.7** (tags/v3.12.7:0b05ead, Oct  1 2024, 03:06:41) [MSC v.1941 64 bit (AMD64)]
** Python executable: C:\Users\qiang.yu\Desktop\ComfyUI_windows_portable\python_embeded\python.exe

Total VRAM 4096 MB, total RAM 32495 MB
pytorch version: 2.5.1+cu124
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA T600 Laptop GPU : cudaMallocAsync
Using pytorch cross attention
[Prompt Server] web root: C:\Users\qiang.yu\Desktop\ComfyUI_windows_portable\ComfyUI\web
C:\Users\qiang.yu\Desktop\ComfyUI_windows_portable\python_embeded\Lib\site-packages\kornia\feature\lightglue.py:44: **FutureWarning: `torch.cuda.amp.custom_fwd(args...)` is deprecated. Please use `torch.amp.custom_fwd(args..., device_type='cuda')` instead.**
  @torch.cuda.amp.custom_fwd(cast_inputs=torch.float32)
### ComfyUI Revision: 2808 [69667290] *DETACHED | Released on '2024-11-04'

Import times for custom nodes:
   0.0 seconds: C:\Users\qiang.yu\Desktop\ComfyUI_windows_portable\ComfyUI\custom_nodes\websocket_image_save.py
   0.3 seconds: C:\Users\qiang.yu\Desktop\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager

and in the UI I got something like this: 
![image](https://github.com/user-attachments/assets/3783830d-f9d9-4e6e-a91a-e42fbb6abda0)
There is no line connecting each note, I do not what the problem is, it seems nobody has this. Maybe the python version?  I only download the manger note, beside, the UI is as it is after installing. I upgraded the chrome to latest. If you have any idea, please let me know. Thank you very much!

**Comment by LukeG89** [comment, marked as answer]
Ehm... The links are hidden 😅 

![visible](https://github.com/user-attachments/assets/fcf8c589-30b4-431b-addd-9e0c559fb386)

**Reply by 565514246** [reply]
Hi Luke,,, thanks for your vision! it works!
